packages/graphql-tower-storage-s3/src/index.js

```diff
@@ -79,9 +80,9 @@ export default class StorageS3 {
     }
   }
 
-  async confirm(key, toKey) {
+  async confirm(key, toKey, format) {
     await this.s3.copyObject({ CopySource: `${this.bucket}/uploader/${key}`, Key: `media/${toKey}` }).promise();
-    await this.transform(`media/${toKey}`, `media/${toKey}_cover`);
+    await this.transform(`media/${toKey}`, `media/${toKey}_cover`, format);
```

Comment:
`await this.transform(media/${toKey}, media/${toKey}_cover, toFormat(sharp(), format));`